modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb

```diff
@@ -15,7 +15,7 @@ def index
         resource = collection_resource
         resource = params[:filter].present? ? resource.find_by(filter_params) : resource
         resource = resource.sort(params[:sort])
-        resource = last_refill_date_filter(resource) if params[0] == '-dispensed date'
+        resource = last_refill_date_filter(resource) if params[:sort].include? '-dispensed_date'
```

Comment:
I think it could be nice to add all the possible parameters to the function description above and ideally link to the spec.